src/strife/f_wipe.h

```diff
@@ -60,4 +60,8 @@ wipe_ScreenWipe
   int		height,
   int		ticks );
 
+// [crispy] Additional fail-safe counter for performing crossfade effect.
+#define CROSSFADETICS  13;
```

Comment:
This may leave now.

Reply:
Left already in https://github.com/fabiangreffrath/crispy-doom/pull/1183/commits/93f36f377cd365963256412b44f22eb2708951dc#diff-29ebd4fc56aace2109023502c0ec43ef2bb46d82cd9f4669eed0003740c1ff8eL63-L65.

Reply:
For some reason I can still see it in the diff. 🤷